.github/workflows/velox_be.yml

```diff
@@ -358,6 +358,13 @@ jobs:
             --local --preset=velox --benchmark-type=h --error-on-memleak --disable-aqe --off-heap-size=20g -s=1.0 --threads=16 --iterations=1 \
           && GLUTEN_IT_JVM_ARGS=-Xmx50G sbin/gluten-it.sh queries-compare \
             --local --preset=velox --benchmark-type=ds --error-on-memleak --off-heap-size=50g -s=30.0 --threads=32 --iterations=1'
+      - name: (To be fixed) TPC-H SF1.0 && TPC-DS SF30.0 Parquet local spark3.2 random kill tasks
```

Comment:
Just curious: why different SF for -H/-DS?

Reply:
@rui-mo Was it because large SF for TPC-H doesn't make much difference?

Reply:
We found different scales of TPC-DS could help uncover more possible issues, but for TPC-H, that could be not needed because it is simpler.